airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/build_image/commands.py

```diff
@@ -17,10 +20,19 @@
     default=False,
     type=bool,
 )
+@click.option(
+    "-a",
+    "--architecture",
+    "build_architectures",
+    help="Architecture for which to build the connector image. If not specified, the image will be built for the local architecture.",
+    multiple=True,
+    default=[LOCAL_MACHINE_TYPE],
+    type=click.Choice(PLATFORM_MACHINE_TO_DAGGER_PLATFORM.keys(), case_sensitive=True),
```

Comment:
❓Would it be better to go off of `BUILD_PLATFORMS` instead? Since those are the architectures we support?